README.md

```diff
@@ -106,6 +106,21 @@ custom:
 ```
 > Note that only relative path is supported at the moment.
 
+If you run into the error `stderr maxBuffer exceeded` while using `serverless-webpack`, there is
+an additional configuration options you can use:
+```yaml
+# serverless.yml
+custom:
+  maxBufferInKb: 1024
```

Comment:
Open to better configuration option names other than `maxBufferInKb`. Perhaps it should be `serverlessWebpackNpmInstallMaxBufferInKb`? That seems ridiculous, but it needs to be name-spaced in the `custom` options somehow.